packages/libs/configuration/resources/plugin-java.md

```diff
@@ -6,7 +6,7 @@ The V3 version of the Java Generator.
 version: ^3.7.0
```

Comment:
Should we set this at `^3.8.4` as this is the recommended `@autorest/core` version?